src/reducer/resource/list/ids.js

```diff
@@ -1,10 +1,57 @@
-import { CRUD_GET_LIST_SUCCESS, CRUD_DELETE_SUCCESS } from '../../../actions/dataActions';
+import { CRUD_GET_MANY_SUCCESS, CRUD_GET_LIST_SUCCESS, CRUD_DELETE_SUCCESS } from '../../../actions/dataActions';
 
-export default resource => (previousState = [], { type, payload, requestPayload, meta }) => {
+const cacheDuration = 10 * 60 * 1000; // ten minutes
+
+/**
+ * Add new records to the pool, and remove outdated ones.
+ *
+ * This is the equivalent of a stale-while-revalidate caching strategy:
+ * The cached data is displayed before fetching, and stale data is removed
+ * only once fresh data is fetched.
+ */
+const addRecords = (newRecords = [], oldRecords) => {
+    const now = new Date();
+    const newRecordsFetchedAt = newRecords.reduce((prev, record) => {
+        prev[record] = now; // eslint-disable-line no-param-reassign
+        return prev;
+    }, []);
+    // remove outdated old records
+    const latestValidDate = new Date();
+    latestValidDate.setTime(latestValidDate.getTime() - cacheDuration);
+    const oldValidRecordIds = oldRecords.fetchedAt
+        ? Object.keys(oldRecords.fetchedAt)
+            .filter(id => oldRecords.fetchedAt[id] > latestValidDate)
+        : [];
+    const oldValidRecords = oldValidRecordIds.reduce((prev, id) => {
+        prev[id] = oldRecords[id]; // eslint-disable-line no-param-reassign
```

Comment:
probably... I need to look at it more deeply. In the meantime, as I want to release 1.1.0, I must revert the change.